### PR TITLE
Crunion's Samurai Armor

### PR DESCRIPTION
**Pull request by carlopault**
image

## Changelog
:cl:
add: The chaplain's wardrobe vendor can now be emagged to acquire a samurai armaments beacon.
add: I have coded an entire samurai set of clothing. This includes a suit, armor, samurai sword, tanto dagger, belt, shoes, etc.
add: The samurai armor allows you to ignore slowdown caused by damage. 
add: The costume vendor in the backstage room can now be used to buy a yakama (samurai clothing). 
soundadd: The samurai armor gives you a special scream.
imageadd: Sprites for everything were done by Crunion (Shazbot)
spellcheck: fixed a typo in the wooden stake's description
/:cl:

## About The Pull Request
Neo helped me code, as always. I coded Crunion's armor because he sprited some books for me.

## Why It's Good For The Game
Chaplain traitor content (or for any traitor who breaks into the chaplain's office).

### CODE REVIEW

**Comment by byandaccount** [comment]
+1 hell yeah, it'd pair great with half the null rod transformations the chaplain has access to already, plus i love me some clothing that gives alternative screams.

**Comment by ghost** [comment]
+1

**Comment by MrEpicSax** [comment]
+1

**Comment by wheezl** [comment]
+1 and accepted but I take all his antag tokens viv promised him

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -306,6 +306,7 @@
 	icon_state = "chapdrobe"
 	product_ads = "Are you being bothered by cultists or pesky revenants? Then come and dress like the holy man!;Clothes for men of the cloth!"
 	vend_reply = "Thank you for using the ChapDrobe!"
+	var/emagged = 0
```
Comment (yoyobatty) [inline code/modules/vending/wardrobes.dm]: emagged is an already existing bitflag, remove this

File: code/modules/vending/wardrobes.dm [hunk 2]
```diff
@@ -324,11 +325,20 @@
...
 					/obj/item/clothing/head/medievaltophat = 3,
 					/obj/item/clothing/suit/chaplainsuit/clownpriest = 1,
-					/obj/item/clothing/head/clownmitre = 1)		
+					/obj/item/clothing/head/clownmitre = 1)
 	premium = list(/obj/item/clothing/suit/chaplainsuit/bishoprobe = 1,
 					/obj/item/clothing/head/bishopmitre = 1)
 	refill_canister = /obj/item/vending_refill/wardrobe/chap_wardrobe
 	payment_department = ACCOUNT_CIV
+
+/obj/machinery/vending/wardrobe/chap_wardrobe/attackby(obj/item/I, mob/user, params)
```
Comment (yoyobatty) [inline code/modules/vending/wardrobes.dm]: there's a proc called `emag_act`

File: code/modules/vending/wardrobes.dm [hunk 3]
```diff
@@ -324,11 +325,20 @@
...
+	if(istype(I, /obj/item/card/emag))
+		if(!emagged)
```
Comment (yoyobatty) [inline code/modules/vending/wardrobes.dm]: use `!(obj_flags & EMAGGED)`

File: code/modules/vending/wardrobes.dm [hunk 4]
```diff
@@ -324,11 +325,20 @@
...
+			emagged = 1
```
Comment (yoyobatty) [inline code/modules/vending/wardrobes.dm]: use `obj_flags |= EMAGGED`

**Comment by OK-Computer-OK** [comment]
+1 because hes a handicapped

**Review by yoyobatty — APPROVED** [review]
Alright, hope it works

**Pull request merged**